library_generation/test/generate_library_unit_tests.sh

```diff
@@ -219,6 +250,133 @@ download_grpc_plugin_failed_with_invalid_arch_test() {
   __assertEquals 1 $((res))
 }
 
+generate_library_success_with_valid_versions() {
+  local os_architecture
+  local destination="google-cloud-alloydb-v1-java"
+  os_architecture=$(__get_os_architecture "${os_type}")
+  cd "${script_dir}/resources/protos"
+  "${script_dir}"/../generate_library.sh \
+    -p google/cloud/alloydb/v1 \
```

Comment:
There are over 300 files added as part of this PR, since the protos under test does not matter, can we use a service that has very few messages? Or even better if we can have a minimum test proto?

Reply:
Sure, do you have a better candidate in mind?

Reply:
Similar to this [test proto](https://github.com/googleapis/sdk-platform-java/blob/main/gapic-generator-java/src/test/proto/wicked.proto) in gapic-generator-java, I guess we'll need test yaml and json as well but they can have minimum content as well.

Reply:
I can copy some proto/yaml/json from [showcase repo](https://github.com/googleapis/gapic-showcase) and only keep the minimum content. WDYT?